Using_Crypt.md

```diff
@@ -26,14 +28,61 @@ Configure Tomcat to use your password from an encrypted or plain text string:
 
 Now that you have a password configured, you can encrypt some value and put the resulting string in your configuration file to be decrypted on the fly:
 
+#### Encrypting Values for Use
+
+You can encrypt values one of two ways.
+
+Method 1, using a plain text encryption password:
+~~~
+$ bin/vault.sh --encrypt MyEncryptionPassword MyPassword
+=========================================================================
+
+  Tomcat Vault
+
+  VAULT_HOME: /path/to/tomcat-vault/lib
+
+  JAVA: java
+
+=========================================================================
+
+Encrypted value: CRYPT::a33AiwJkF4dMx9Uq9oxElYT6LdjXLJxf
+~~~
+
+Method 2, using an encryption password which is stored in the vault:
+
 ~~~
-$ java -cp lib/tomcat-juli.jar:lib/tomcat-util.jar:lib/tomcat-vault.jar org.apache.tomcat.vault.util.PropertySourceVault MyEncryptionPassword MyPassword
-Encrypted value: CRYPT::9kofG2Sd1qUdDT0+XIKx+rzjsAZulJJQ
+$ bin/vault.sh --keystore /path/to/vault.keystore --keystore-password my_password123 --alias my_vault --enc-dir /path/to/vault/encryption_dir/ --iteration 44 --salt 1234abcd --encrypt VAULT::my_block::my_encryption_password:: MyPassword
```

Comment:
The two options `--iteration 44` and `--salt 1234abcd` are not required for this command. Because these options are used for computing a masked keystore password but this command does not output such masked password. So, I think the two options can be removed from this example.

Reply:
Fixed.